### ISSUE TITLE
New submissions for Tue, 19 Jul 22

### ISSUE THREAD

**Issue by imjunwei** [imjunwei/ocr-daily-paper#242]
### Real-time End-to-End Video Text Spotter with Contrastive Representation  Learning
 - **Authors:** Wejia Wu, Zhuang Li, Jiahong Li, Chunhua Shen, Hong Zhou, Size Li, Zhongyuan Wang, Ping Luo
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2207.08417
 - **Pdf link:** https://arxiv.org/pdf/2207.08417
 - **Abstract**
 Video text spotting(VTS) is the task that requires simultaneously detecting, tracking and recognizing text in the video. Existing video text spotting methods typically develop sophisticated pipelines and multiple models, which is not friend for real-time applications. Here we propose a real-time end-to-end video text spotter with Contrastive Representation learning (CoText). Our contributions are three-fold: 1) CoText simultaneously address the three tasks (e.g., text detection, tracking, recognition) in a real-time end-to-end trainable framework. 2) With contrastive learning, CoText models long-range dependencies and learning temporal information across multiple frames. 3) A simple, lightweight architecture is designed for effective and accurate performance, including GPU-parallel detection post-processing, CTC-based recognition head with Masked RoI. Extensive experiments show the superiority of our method. Especially, CoText achieves an video text spotting IDF1 of 72.0% at 41.0 FPS on ICDAR2015video, with 10.5% and 32.0 FPS improvement the previous best method. The code can be found at github.com/weijiawu/CoText.
### 2D Self-Organized ONN Model For Handwritten Text Recognition
 - **Authors:** Hanadi Hassen Mohammed, Junaid Malik, Somaya Al-Madeed, Serkan Kiranyaz
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Artificial Intelligence (cs.AI)
 - **Arxiv link:** https://arxiv.org/abs/2207.08139
 - **Pdf link:** https://arxiv.org/pdf/2207.08139
 - **Abstract**
 Deep Convolutional Neural Networks (CNNs) have recently reached state-of-the-art Handwritten Text Recognition (HTR) performance. However, recent research has shown that typical CNNs' learning performance is limited since they are homogeneous networks with a simple (linear) neuron model. With their heterogeneous network structure incorporating non-linear neurons, Operational Neural Networks (ONNs) have recently been proposed to address this drawback. Self-ONNs are self-organized variations of ONNs with the generative neuron model that can generate any non-linear function using the Taylor approximation. In this study, in order to improve the state-of-the-art performance level in HTR, the 2D Self-organized ONNs (Self-ONNs) in the core of a novel network model are proposed. Moreover, deformable convolutions, which have recently been demonstrated to tackle variations in the writing styles better, are utilized in this study. The results over the IAM English dataset and HADARA80P Arabic dataset show that the proposed model with the operational layers of Self-ONNs significantly improves Character Error Rate (CER) and Word Error Rate (WER). Compared with its counterpart CNNs, Self-ONNs reduce CER and WER by 1.2% and 3.4 % in the HADARA80P and 0.199% and 1.244% in the IAM dataset. The results over the benchmark IAM demonstrate that the proposed model with the operational layers of Self-ONNs outperforms recent deep CNN models by a significant margin while the use of Self-ONNs with deformable convolutions demonstrates exceptional results.
### Transcribing Medieval Manuscripts for Machine Learning
 - **Authors:** Estelle Guéville, David Joseph Wrisley
 - **Subjects:** Digital Libraries (cs.DL)
 - **Arxiv link:** https://arxiv.org/abs/2207.07726
 - **Pdf link:** https://arxiv.org/pdf/2207.07726
 - **Abstract**
 In the early twentieth century, many scholars focused on the preparation of editions and translations of texts previously available only to the few specialists able to read archaic hands and privileged enough to travel to work in person with them in manuscript. Valuable scholarship in its own right, the preparation of these editions and translations for particular texts deemed important enough to justify the effort and time, laid the foundation for generations of scholarship in medieval studies. On the other hand, for many materials in historical archival collections, including already digitised collections, medievalists have only had the time to create partial transcriptions, if any at all. Access to textual material from the medieval period has increased greatly in recent years with digitisation, and we are able to imagine many new research projects in decades to come. What challenges do new frontiers of automation in the archives raise with respect to medieval studies and in particular to the ways we transcribe? In this article, we argue that if medievalists hope to pursue the kinds of analysis that goes on in advanced computational research, we will need new kinds of transcriptions, intentionally theorized not only for human reading, but also for machine processing. We already have mature methods for remediating generations of editions of medieval works such as Optical Character Recognition (OCR), but we can ask ourselves if these are the kinds of text we want to use for future computational analysis. We suggest instead that one way forward is by going back to the scriptorium.
### Printable Flexible Robots for Remote Learning
 - **Authors:** Savita V. Kendre, Gus. T. Teran, Lauryn Whiteside, Tyler Looney, Ryley Wheelock, Surya Ghai, Markus P. Nemitz
 - **Subjects:** Robotics (cs.RO); Computers and Society (cs.CY)
 - **Arxiv link:** https://arxiv.org/abs/2207.07729
 - **Pdf link:** https://arxiv.org/pdf/2207.07729
 - **Abstract**
 The COVID-19 pandemic has revealed the importance of digital fabrication to enable online learning, which remains a challenge for robotics courses. We introduce a teaching methodology that allows students to participate remotely in a hands-on robotics course involving the design and fabrication of robots. Our methodology employs 3D printing techniques with flexible filaments to create innovative soft robots; robots are made from flexible, as opposed to rigid, materials. Students design flexible robotic components such as actuators, sensors, and controllers using CAD software, upload their designs to a remote 3D printing station, monitor the print with a web camera, and inspect the components with lab staff before being mailed for testing and assembly. At the end of the course, students will have iterated through several designs and created fluidically-driven soft robots. Our remote teaching methodology enables educators to utilize 3D printing resources to teach soft robotics and cultivate creativity among students to design novel and innovative robots. Our methodology seeks to democratize robotics engineering by decoupling hands-on learning experiences from expensive equipment in the learning environment.
### Spatial Distribution of Solar PV Deployment: An Application of the  Region-Based Convolutional Neural Network
 - **Authors:** Serena Y. Kim, Koushik Ganesan, Crystal Soderman, Raven O'Rourke
 - **Subjects:** Computers and Society (cs.CY)
 - **Arxiv link:** https://arxiv.org/abs/2207.08287
 - **Pdf link:** https://arxiv.org/pdf/2207.08287
 - **Abstract**
 This paper presents a comprehensive analysis of the social and environmental determinants of solar photovoltaic (PV) deployment rates in Colorado, USA. Using 652,795 satellite imagery and computer vision frameworks based on a convolutional neural network, we estimated the proportion of households with solar PV systems and the roof areas covered by solar panels. At the census block group level, 7% of Coloradan households have a rooftop PV system, and 2.5% of roof areas in Colorado are covered by solar panels as of 2021. Our machine learning models predict solar PV deployment based on 43 natural and social characteristics of neighborhoods. Using four algorithms (Random Forest, CATBoost, LightGBM, XGBoost), we find that the share of Democratic party votes, hail risks, strong wind risks, median home value, and solar PV permitting timelines are the most important predictors of solar PV count per household. In addition to the size of the houses, PV-to-roof area ratio is highly dependent on solar PV permitting timelines, proportion of renters and multifamily housing, and winter weather risks. We also find racial and ethnic disparities in rooftop solar deployment. The average marginal effects of median household income on solar deployment are lower in communities with a greater proportion of African American and Hispanic residents and are higher in communities with a greater proportion of White and Asian residents. In the ongoing energy transition, knowing the key predictors of solar deployment can better inform business and policy decision making for more efficient and equitable grid infrastructure investment and distributed energy resource management.
### Symmetrized Robust Procrustes: Constant-Factor Approximation and Exact  Recovery
 - **Authors:** Tal Amir, Shahar Kovalsky, Nadav Dym
 - **Subjects:** Machine Learning (cs.LG); Computational Geometry (cs.CG); Computer Vision and Pattern Recognition (cs.CV); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2207.08592
 - **Pdf link:** https://arxiv.org/pdf/2207.08592
 - **Abstract**
 The classical $\textit{Procrustes}$ problem is to find a rigid motion (orthogonal transformation and translation) that best aligns two given point-sets in the least-squares sense. The $\textit{Robust Procrustes}$ problem is an important variant, in which a power-1 objective is used instead of least squares to improve robustness to outliers. While the optimal solution of the least-squares problem can be easily computed in closed form, dating back to Sch\"onemann (1966), no such solution is known for the power-1 problem. In this paper we propose a novel convex relaxation for the Robust Procrustes problem. Our relaxation enjoys several theoretical and practical advantages: Theoretically, we prove that our method provides a $\sqrt{2}$-factor approximation to the Robust Procrustes problem, and that, under appropriate assumptions, it exactly recovers the true rigid motion from point correspondences contaminated by outliers. In practice, we find in numerical experiments on both synthetic and real robust Procrustes problems, that our method performs similarly to the standard Iteratively Reweighted Least Squares (IRLS). However the convexity of our algorithm allows incorporating additional convex penalties, which are not readily amenable to IRLS. This turns out to be a substantial advantage, leading to improved results in high-dimensional problems, including non-rigid shape alignment and semi-supervised interlingual word translation.
## Keyword: Handwriting
There is no result 
## Keyword: Scene Text
There is no result 
## Keyword: Text Segmentation
There is no result 
## Keyword: TextSpotter
There is no result 
### Real-time End-to-End Video Text Spotter with Contrastive Representation  Learning
 - **Authors:** Wejia Wu, Zhuang Li, Jiahong Li, Chunhua Shen, Hong Zhou, Size Li, Zhongyuan Wang, Ping Luo
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2207.08417
 - **Pdf link:** https://arxiv.org/pdf/2207.08417
 - **Abstract**
 Video text spotting(VTS) is the task that requires simultaneously detecting, tracking and recognizing text in the video. Existing video text spotting methods typically develop sophisticated pipelines and multiple models, which is not friend for real-time applications. Here we propose a real-time end-to-end video text spotter with Contrastive Representation learning (CoText). Our contributions are three-fold: 1) CoText simultaneously address the three tasks (e.g., text detection, tracking, recognition) in a real-time end-to-end trainable framework. 2) With contrastive learning, CoText models long-range dependencies and learning temporal information across multiple frames. 3) A simple, lightweight architecture is designed for effective and accurate performance, including GPU-parallel detection post-processing, CTC-based recognition head with Masked RoI. Extensive experiments show the superiority of our method. Especially, CoText achieves an video text spotting IDF1 of 72.0% at 41.0 FPS on ICDAR2015video, with 10.5% and 32.0 FPS improvement the previous best method. The code can be found at github.com/weijiawu/CoText.
